lib/web/machineid.go

```diff
@@ -21,6 +21,7 @@ import (
 	"strconv"
 	"time"
 
+	yaml "github.com/ghodss/yaml"
```

Comment:
I can't keep track of which YAML package to use anymore, and I see this package is already a dependency of `lib/web` so I think you got it right. Worth noting that `ghodss/yaml` seems less common in our codebase though.

```
teleport on master [$!] ➜ ag --go -Q --count 'gopkg.in/yaml' | wc -l
      58

teleport on master [$!] ➜ ag --go -Q --count 'ghodss/yaml' | wc -l
      21
```